src/ServerDaemon.chpl

```diff
@@ -645,8 +645,11 @@ module ServerDaemon {
                                               "<<< %s took %.17r sec".format(cmd, elapsedTime));
                 }
                 if (trace && memTrack) {
+                    var memUsed = getMemUsed():uint * numLocales:uint;
+                    var pctMemUsed = ((memUsed:real/memMax:real)*100):int;
                     sdLogger.info(getModuleName(),getRoutineName(),getLineNumber(),
-                        "bytes of memory used after command %t".format(getMemUsed():uint * numLocales:uint));
+                        "bytes of memory %t pct of max memory %t used after command".format(memUsed,
```

Comment:
We could have this print out a `%` after the `pct of max memory`. You just gotta double up because % is used to escape characters. So something like
```chapel
"bytes of memory %t and pct of max memory %t%% used after command".format(memUsed, pctMemUsed));
```

Reply:
Done

![image](https://user-images.githubusercontent.com/10785153/219428766-fa0a2307-d7a5-4811-a6d1-7620a83a99a7.png)

Reply:
@hokiegeek2 - I am not seeing this added. Otherwise looks fine.